var/spack/repos/builtin/packages/migraphx/package.py

```diff
@@ -93,7 +94,9 @@ def url_for_version(self, version):
     )
 
     patch("0001-Adding-nlohmann-json-include-directory.patch", when="@3.9.0:")
-    patch("0002-restrict-python-2.7-usage.patch", when="@3.9.0:")
+    # Restrict Python 2.7 usage to fix the issue 24429
```

Comment:
Can you add the URL of the issue:
```suggestion
    # Restrict Python 2.7 usage to fix <url>
```
?

Reply:
Added the URL.